packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -424,4 +441,57 @@ export class GCFBootstrapper {
       });
     }
   }
+
+  /*
+   * Setting the process.env.WEBHOOK_TMP environment variable indicates
+   * that the webhook payload should be written to a tmp file in Cloud
+   * Storage. This allows us to circumvent the 100kb limit on Cloud Tasks.
+   *
+   * @param body
+   */
+  private async maybeWriteBodyToTmp(body: string): Promise<string> {
```

Comment:
Are there docs about how the bots work? Would you mind adding something short saying, "Payloads can/are stored in Cloud Storage. ..."?

Reply:
👍 I will figure out a place to mention this.

Reply:
@tbpg I created a tracking ticket, there's an architecture diagram that we simply need to update. I think we should wait until we've proven this approach out:

https://github.com/googleapis/repo-automation-bots/issues/986